### ISSUE TITLE
Merging recognized faces of different people is cumbersome and error prone

### ISSUE THREAD

**Issue by csware** [nextcloud/photos#1505]
**Describe the bug**
Install the face recognition. Wehn trying to merge detected persons, the list is constanly updating and it is highly likely that the order changes and ylou click on the 'wrong' person.

**To Reproduce**
1. Go to `/apps/photos/faces/`
2. Select a person
3. Open the menu and select 'merge'

**Expected behavior**
The list is static, ordered by the number of associated fotos, and the images are loaded on an best-.effort basis.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser Firefox
 - Version 107

**Comment by bpmartin20** [comment]
I agree with the first paragraph particularly. This may be related to the currency of the browser cache. I often find I have to clear the browser cache (Firefox 102.2.0esr) or the selection window doesn't populate at all, and after I do the images come up randomly and keep resorting as they do so. I don't mind them popping in as the cache populates -- it's the resorting that causes me to click the wrong one. I also wish we had an "Undo" option, so at least I could undo the damage.

**Comment by simonspa** [comment]
I am having the same issue - in addition, merging people only works for *one* person at a time. If I go back to faces and click a different person and try to merge there, the dialog does not open. It requires a force-reload of the entire page (including all faces, veeeery slow) to work for the next person.

(cc) @marcelklehr

**Comment by Sockolet** [comment]
> I am having the same issue - in addition, merging people only works for _one_ person at a time. If I go back to faces and click a different person and try to merge there, the dialog does not open. It requires a force-reload of the entire page (including all faces, veeeery slow) to work for the next person.
> 
> (cc) @marcelklehr

same here. Also my CPU goes to 100% for all cores.
